image/bin/my_init

```diff
@@ -385,16 +409,16 @@ parser = argparse.ArgumentParser(description='Initialize the system.')
 parser.add_argument('main_command', metavar='MAIN_COMMAND', type=str, nargs='*',
                     help='The main command to run. (default: runit)')
 parser.add_argument('--enable-insecure-key', dest='enable_insecure_key',
-                    action='store_const', const=True, default=False,
+                    action='store_const', const=True, default=ENABLE_INSECURE_KEY,
```

Comment:
I would inline these functions, so that we only have one variable representing these states, not two separate where one might be overwritten.